src/_guides/libraries/testing.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: "Dart Testing"
+description: "How to test Dart, Flutter, and 'Dart for the Web' applications?"
+---
+
+Software testing, an important part of app development, helps verify that
+your app is working correctly before unleashing it on your users.
```

Comment:
nit: depending on where on the informal <=> formal spectrum this guide is, "unleashing" may or may not sound appropriate.